x-pack/legacy/plugins/siem/server/lib/index_fields/elasticsearch_adapter.ts

```diff
@@ -51,6 +51,23 @@ export class ElasticsearchIndexFieldAdapter implements FieldsAdapter {
   }
 }
 
+const missingFields = [
+  {
+    name: '_id',
+    type: 'string',
+    searchable: true,
+    aggregatable: true,
```

Comment:
Intuitively, it feels like the `_id` field would be `aggregatable: false`, because of the top of my head, I can't think of a case where presenting a count of `_id` to users makes sense. Consider changing this to `false` if you agree.